vent/io/devices/base.py

```diff
@@ -272,21 +247,13 @@ def __init__(self, offset, mask, values):
                 self._mask = mask
                 self._values = values
 
-            def offset(self) -> int:
-                """ Returns the position of the field's LSB in the register. e.g., mask = self._mask << self._offset."""
-                return self._offset
-
-            def info(self) -> list:
-                """ Returns a list containing the ValueField's offset, mask, and a tuple of possible values """
-                return [self._offset, self._mask, self._values]
-
-            def unpack(self, cfg) -> OrderedDict:
+            def unpack(self, cfg):
                 """ Extracts the ValueField's setting from cfg & returns the result in a human readable form.
 
                 Args:
                     cfg (int): An integer representing a possible configuration value for the register
                 """
-                return OrderedDict(map(reversed, self._values.items()))[self.extract(cfg)]
+                return dict(map(reversed, self._values.items()))[self.extract(cfg)]
```

Comment:
don't really understand what this is doing but i trust u on it

Reply:
Like in general, or why that change in particular? Because now that I'm looking at it again, that was a stupid change to make and I'm changing it back.

never trust